Android/Shared/ITMCopyAssets.gradle

```diff
@@ -0,0 +1,34 @@
+import java.nio.file.Files
```

Comment:
This file's name is inconsistent with ITMAppConfig.gradle's filename. That file is named that because it is a file that deals with ITMAppConfig.json. I added the "gen" prefix to the task name because I felt that was appropriate. I think that either this file should be named ITMAssets.gradle (with copyITMAssets still as the task name), or ITMAppConfig.gradle should be renamed to ITMGenAppConfig.gradle. (I think that renaming this file is preferable.)

Reply:
renamed to ITMAssets.gradle